app_flutter/lib/widgets/task_grid.dart

```diff
@@ -81,6 +441,23 @@ class _TaskGridState extends State<TaskGrid> {
   // lattice matrix each time the task grid has to update, regardless of whether
   // we've received new data or not.
 
+  TaskGridFilter filter;
```

Comment:
dartdoc?

Reply:
Made private as it is just a cached copy for triggering state changes.